src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+    <div class="stars">
+      <div class="stars--0">
```

Comment:
```suggestion
      <div class="stars stars--0">
```
Add this common class for each row